posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -126,6 +126,32 @@ def where(self):
                 ),
             )
 
+        if self.query.searchQuery:
+            # TODO: Refine this so it only searches the frames inside $exception_list
+            props_to_search = ["$exception_list", "$exception_stack_trace_raw", "$exception_type", "$exception_message"]
+            or_exprs: list[ast.Expr] = []
+            for prop in props_to_search:
+                or_exprs.append(
+                    ast.CompareOperation(
+                        op=ast.CompareOperationOp.Gt,
```

Comment:
Not a concern for right now but maybe we could consider some degree of fuzzy matching in future

Reply:
nice 👌